core/platform/taskqueue/dev_mode_taskqueue_services_test.py

```diff
@@ -77,7 +88,12 @@ def test_task_handler_will_create_the_correct_post_request(self):
             'X-AppEngine-Fake-Is-Admin': '1',
             'method': 'POST'
         }
-        def mock_post(url, json, headers, timeout):
+        def mock_post(
+                url: str,
+                json: Dict[str, Any],
```

Comment:
Missing TODO/Justification of why this usage of `Dict[str, Any]` is valid.

Reply:
Moved to doc

Reply:
Done